core/base/ftmTree/FTMTreeUtils.cpp

```diff
@@ -1,4 +1,17 @@
 #include <FTMTree.h>
+#include <FTMTreeUtils.h>
+#include <iostream>
+
+void myPause() {
+  std::cout << "pause" << std::endl;
+  int temp;
+  std::cin >> temp;
+}
+
+void printTreesStats(std::vector<ftm::FTMTree_MT *> &trees) {
```

Comment:
thanks for making sure that this function is defined in the `ftm` name space.

Reply:
Ok! Done in the next commit.